config/main.py

```diff
@@ -1048,9 +1036,6 @@ def reload(db, filename, yes, load_sysinfo, no_service_restart):
     if not yes:
         click.confirm(message, abort=True)
 
-    global child_pid
```

Comment:
You _are_ setting `child_pid` in this function, and it is a global, so I think you should have kept this line.